src/dmd/backend/bcomplex.d

```diff
@@ -11,7 +11,7 @@ module dmd.backend.bcomplex;
 
 public import dmd.root.longdouble : targ_ldouble = longdouble;
 import core.stdc.math : fabs, fabsl, sqrt;
-version(CRuntime_Microsoft)
+static if (real.sizeof == 8)
```

Comment:
also WTF!

This condition is true (verified with `pragma(msg)`on apple arm64 and fails as is. 
If I remove the line and unconditionally `import dmd.root.longdouble` then it works!??!?!

Reply:
Version conditions are expanded first, then static ifs later in semantic.  You're probably unwittingly making another version statement depend on this static if, which is bad.

Reply:
Hmm, let's just see if an unconditional import works for all systems.